lib/pages/user_regattas.dart

```diff
@@ -46,12 +50,13 @@ class _UserRegattasPageState extends State<UserRegattasPage> {
             controller: teamController,
           ),
           ElevatedButton(
-            onPressed: () => Navigator.pushNamed(
-              context,
-              RacePage.route,
-              arguments: RacePageArguments(
-                  eventIdController.text, teamController.text),
-            ),
+            onPressed: () async {
+            // TODO move team score initialization to team creation flow
+              await dbRef.initializeScoreForTeam(eventIdController.text, teamController.text);
+              Navigator.pushNamed(context, RacePage.route,
```

Comment:
I don't like this. It would mean if someone leaves race page and then go back his team score is reset

Reply:
true but it is ok as this is just a temporary way of initialization